apps/opencs/view/render/scenewidget.hpp

```diff
@@ -61,8 +79,17 @@ namespace CSVRender
 
         protected:
 
+            CompositeOsgRenderer* mRenderer;
             osg::ref_ptr<osgViewer::View> mView;
             osg::ref_ptr<osg::Group> mRootNode;
+            std::mutex _osgMutex;
```

Comment:
This looks like OSG member variable naming style. For OpenMW code, mMemberVariable is preferred.

Reply:
Good point, but I won't fix this yet, because this is a part of merging osgQOpenGLWidget code into SceneWidget. See the comment above https://github.com/OpenMW/openmw/pull/3031#discussion_r531452666

> Would it perhaps be better to develop osgQOpenGLWidget and use it as member widget of SceneWidget?